README.md

```diff
@@ -84,6 +84,12 @@ Note that if you have not enabled TagBot, no release will be made at all, and so
 
 The Julia package manager **does not** rely on git tags and GitHub releases. However, Registrator will generate a `git tag` command for you to optionally create a corresponding tag with your package version, or you can use TagBot as is mentioned above.
 
+### Note on documentation build
+
+The docs for your project will be automatically built by [DocumentationGenerator.jl](https://github.com/JuliaDocs/DocumentationGenerator.jl). Please see that repo for details. Your docs should show up at pkg.julialang.org/docs.
+
+By default, whatever is in `docs/make.jl` will be built, or it will fall back to using your README.md.
```

Comment:
I would phrase it a bit less informally. I made a suggestion below but feel free to modify as you like.

```suggestion
By default, `docs/make.jl` will be run to build the docs. If that is missing, the `README.md` will be used instead.
```

Reply:
I think those changes look great!